### DISCUSSION TITLE
Riptroduzione di playlist musicali

### DISCUSSION THREAD

**Discussion by texindustries** [indomus/forum#3155] [HUB: Home Assistant]
Ciao ragazzi
un post nato dal fatto che son un po tirchio nel pagare spotify premium... Volevo capire se c'è un modo easy, uno script per home assistant che possa andarmi a riprodurre degli mp3 nella cartella media del home assistant (che in realtà è una cartella in rete, visibile dal sistema).
mi farebbe comodo uno script per creare un ambiente "nanna" per mia figlia, ovvero eseguo lo script, mi abbassa le luci al tot% (già pronto) e riproduca su google home in sequenza degli mp3 di relax.
C'è un modo semplice per farli riprodurre uno dopo l'altro?

grazie in anticipo per tutti i suggerimenti

**Comment by KC-inDomus** [comment]
chioso: non conviene utilizzare un oggetto software "proprio" per la riproduzione degli mp3, e poi semmai integrare quello a home assistant, piuttosto che usare quest'ultimo per un mestiere che non è il suo?

**Reply by texindustries** [reply]
beh si se esite... qualche consiglio come software? VNC negli smartphone leggono prefettamente la playlist ma non li trasmettono sugli smart speaker (solo su smart tv o chromcast)...

**Reply by KC-inDomus** [reply]
ne esistono quanti ne vuoi. prendi spunto dalle integrazioni di Home Assistant https://www.home-assistant.io/integrations/#media-player e vai alla ricerca. valuta per esempio Music Player Daemon (MPD), ma troppi ce ne sono

**Reply by texindustries** [reply]
Beh con questi sistemi ok... Mi ci vuole come minimo un Raspberry dove giri il deamon, oggi come oggi ho home assistant OS, non su docker... Comunque ok questa è una soluzione facilmente percorribile, speravo qualcosa di più eseguibile con uno script

**Reply by KC-inDomus** [reply]
eh, vedi se su home assistant os c'è un add-on che possa fungere allo scopo
